.github/workflows/push.yml

```
@@ -281,11 +327,33 @@ jobs:
+          echo APP="${AFFECTED_PROJECTS}" >> "$GITHUB_ENV"
+
+      - name: Login to ECR
+        uses: docker/login-action@v3
+        id: login-ecr
+        with:
+          registry: ${{ env.DOCKER_REGISTRY }}
+          username: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+      - name: Set environment variables for build
+        run: |
```

actionlint (1.7.4)

property "registry" is not defined in object type {}

(expression)